### ISSUE TITLE
After updating to "3.0.1" the overlay is not centered

### ISSUE THREAD

**Issue by jmesa-sistel** [Mawi137/ngx-image-cropper#254]
Hi, after updating to "3.0.1" the overlay is not centered

```
<image-cropper #cropper [imageBase64]="imageBase64" format="png" (imageCropped)="imageCropped($event)" (imageLoaded)="imageLoaded()" [cropper]="cropperPosition"
                    (loadImageFailed)="loadImageFailed()" [alignImage]="'center'" [autoCrop]="false"></image-cropper>

```
![image](https://user-images.githubusercontent.com/22615079/72880257-cf917e80-3cfe-11ea-9e6c-be2db9244d28.png)

I think you have to add:
`[style.margin-left]="alignImage === 'center' ? marginLeft : null"`
in [image-cropper.component.html#L14](https://github.com/Mawi137/ngx-image-cropper/blob/master/projects/ngx-image-cropper/src/lib/component/image-cropper.component.html#L14)

**Comment by Mawi137** [comment]
Hi yes you are correct, I’ll fix it ASAP, which will be on Sunday because I don’t have access to my laptop till then :)

**Comment by Mawi137** [comment]
Fixed as of version `3.0.2`!
